tests/test_unittest/test_storage.py

```diff
@@ -14,63 +14,53 @@
 #
```

Comment:
Can this file be split into more taxonomic parts, possibly under a `storage` subdirectory? It is hard for me to determine whether all the features of Storage are tested because the file is so large.